nautobot/extras/tests/example_jobs/test_job_hook_receiver.py

```diff
@@ -0,0 +1,13 @@
+from nautobot.extras.jobs import JobHookReceiver
+from nautobot.dcim.models import Site
+
+
+class TestJobHookReceiverLog(JobHookReceiver):
+    def run(self, data, commit):
+        object_change = data["object_change"]
+        self.log_success(object_change.changed_object.name)
+
+
+class TestJobHookReceiverChange(JobHookReceiver):
```

Comment:
I observed that neither of these are implemented a `receive_jobhook()` method. Shouldn't this be an error? Otherwise if they're just being used to test basic characteristics of the classes themselves, can you please indicate that in some docstrings? Thanks!

Reply:
That was an oversight. Thanks for catching it!